File: 02.PYTHON/pg-166-169/pg_166_15.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("READ A THREE-DIGITS INTEGER AND DETERMINE THE SUM OF ITS DIGITS");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("");

num = int(input("Write a three-digit integer: "));

if(num < 0):
    num *= -1;

if(num > 100 and num < 999):
    dig3 = num % 10;
    dig2 = num // 10 % 10;
    dig1 = num // 100 % 10;

    sum = dig1 + dig2 + dig3;

    print("The sum of all digits is: ", sum);
else:
    print("The written number doesn't have three digits. Please try again!");

File: 01.JAVA/pg-166-169/pg_166_15.java
import java.util.Scanner;
public class pg_166_15
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num;
        int dig1, dig2, dig3;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ A THREE-DIGITS INTEGER AND DETERMINE THE SUM OF ITS DIGITS");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");

        System.out.print("Write a three-digits integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1; 
        }

        if(num > 100 && num < 999)
        {
            dig3 = num % 10;
            dig2 = num % 100 / 10;
            dig1 = num / 100 % 10;

            int sum = dig1 + dig2 + dig3;

            System.out.println("The sum of all digits is: " + sum);
        }
        else
        {
            System.out.println("The written number doesn't have three digits. Please try again!");
        }

        

    }
}